src/main/java/li/cil/tis3d/common/LocalAPI.java

```diff
@@ -0,0 +1,13 @@
+package li.cil.tis3d.common;
+
+import li.cil.tis3d.api.CommonAPI;
+import li.cil.tis3d.api.ClientAPI;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+public class LocalAPI {
```

Comment:
"Local" invokes the idea of "on this machine" for me, e.g. as in related to networking. Something like "InternalAPI" -- or even just "API"? -- would sound more clear to me at least? Other suggestions?

Reply:
`API` sounds nice and short! Will use it instead if you're okay with that :) "internal" sounds good too;

Reply:
I'd say just `API` then; was thinking about the possible confusion with the old `API` class, but it moved packages, so that should be fine.